attributes/default.rb

```diff
@@ -206,6 +206,7 @@
 default['cassandra']['opscenter']['server']['package_name'] = 'opscenter'
 default['cassandra']['opscenter']['server']['port'] = '8888'
 default['cassandra']['opscenter']['server']['interface'] = '0.0.0.0'
+default['cassandra']['opscenter']['server']['authentication'] = 'true' #default set to true
```

Comment:
Since this is currently off for anyone using the cookbook, and the server default is false, this should probably default to `false`.  This should also be a boolean value rather than a string.